package.json

```diff
@@ -15,7 +15,7 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/openmrs/openmrs-web-style-referenceapplicaiton.git"
+    "url": "https://github.com/openmrs/openmrs-web-style-referenceapplication"
   },
```

Comment:
Wouldn't it be better to add the `.git` at the end of the url like how the previous URL was structured

Reply:
Okay, fixed it.